source/diagnostics.tex

```diff
@@ -1192,7 +1201,6 @@
 
   // \ref{syserr.errcode.nonmembers} non-member functions:
   error_code make_error_code(errc e) noexcept;
-  bool operator<(const error_code& lhs, const error_code& rhs) noexcept;
 
   template <class charT, class traits>
     basic_ostream<charT, traits>&
```

Comment:
Ah, I think I did it that way because `make_error_code(errc e)` is also in both places, and `operator<<` is grouped with that (both before and after my change) in [syserr.errcode.nonmembers], but `operator<<` was missing from the header synopsis, and after my change is not in [syserr.errcondition.nonmembers].

If I remove `operator<<` from the class synopsis I should also remove `make_error_code(errc)`. I think I prefer not to remove either.

I moved the relational operators to be alongside the equality operators, which are only in the header synopsis not the class synopses. That's because the equality ops are in [syserr.compare] which is at the same level as `error_code` and `error_condition`, rather than being a "non-member functions" subclause.